src/components/layout/Header.tsx

```
@@ -20,15 +20,25 @@ import {
 interface HeaderProps {
   showMenu?: boolean;
   showSearchBar?: boolean;
   showLanguage?: boolean;
 }
+interface MenuOption {
+  name: string;
+  icon?: React.ReactElement; // icon can be a React node
+  onClick?: () => void; // optional click handler
+}
+
+interface MenuItem {
```

Biome

[error] Shouldn't redeclare 'MenuItem'. Consider to delete it or rename it.

'MenuItem' is defined here:


(lint/suspicious/noRedeclare)